pilot/pkg/networking/core/v1alpha3/listener_waypoint.go

```diff
@@ -144,39 +139,21 @@ func (lb *ListenerBuilder) buildWaypointInboundTerminateConnect(svcs map[host.Na
 	}
 
 	// it's possible for us to hit this listener and target a Pod directly; route through the inbound-pod internal listener
```

Comment:
Does this comment (pod internal listener) need to be updated?

Reply:
Updated I think.